Gems/ROS2/Code/Source/Lidar/LidarRegistrarSystemComponent.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "LidarRegistrarSystemComponent.h"
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace ROS2
+{
+    LidarRegistrarSystemComponent::LidarRegistrarSystemComponent()
+    {
+        if (LidarRegistrarInterface::Get() == nullptr)
```

Comment:
uber nit: `if (!LidarRegistrarInterface::Get())` style is generally preferred

Reply:
Thanks! Fixed.